invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/deposit.less

```diff
@@ -129,3 +128,29 @@
 .item div.description:not(.license-item-description) {
   display: initial !important;
 }
+
+.deposits-community-header {
+  margin: -20px 0px 20px 0px;
+  background-color: #ededed;
+  height: 65px;
```

Comment:
usage of fixed units is not the best practice, especially in the theme which relies on relative units